apps/aevm/src/aevm_eeevm.erl

```diff
@@ -778,202 +782,170 @@ loop(StateIn) ->
                     %% default to zero if they extend past the limits.
                     %% The byte is right-aligned (takes the lowest
                     %% significant place in big endian).
-                    State1 = push_n_bytes_from_cp(OP-?PUSH1+1, State0),
-                    next_instruction(OP, State, State1);
+                    push_n_bytes_from_cp(CP, 1, State, State0);
                 ?PUSH2 ->
                     %% 0x61 PUSH1 δ=0 α=1
                     %% Place 2 byte item on stack.
-                    State1 = push_n_bytes_from_cp(OP-?PUSH1+1, State0),
-                    next_instruction(OP, State, State1);
+                    push_n_bytes_from_cp(CP, 2, State, State0);
```

Comment:
No problem though just asking: for PUSH5 onwards I see the derived computation (OP - ThisOp + 1) kept. Is it intended to remove derived computation for PUSH1...PUSH4?

Reply:
Yes. Push 1-4 and 32 are most used, but 32 is expensive anyway so I think it gives a bit of a speedup to do one calculation less for push 1 to 4. Future version will make all of them more efficient.